src/domUpdates.js

```diff
@@ -429,5 +419,5 @@ export {
   renderActiveTag,
   displayTaggedRecipes,
   renderRecipesOfInterest,
-  enableScrollPitchText
+  enableScrollPitchText,
```

Comment:
i guess we don't need this comma?

Reply:
fixed in merge conflict resolution, done now